patient-search/ui/src/pages/advancedSearch/components/InvestigationResults.tsx

```diff
@@ -108,6 +108,22 @@ export const InvestigationResults = ({
                         style={{ wordBreak: 'break-word', cursor: 'pointer' }}>
                         {name}
                     </p>
+                    <p
+                        onClick={async () => {
```

Comment:
Better have this function defined with a specific name above and reference here.